config.go

```diff
@@ -1173,6 +1166,24 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
 			confFile = "litecoin"
 		}
 
+		// If all of RPCUser, RPCPass, ZMQBlockHost, and ZMQTxHost are
+		// set, we assume those parameters are good to use.
+		if conf.ZMQPubRawBlock != "" && conf.ZMQPubRawTx != "" {
+			if conf.RPCUser != "" && conf.RPCPass != "" {
+				return nil
+			}
+			// Try to get RPCUser and RPCPass from an auth cookie
+			rpcUser, rpcPass, err :=
```

Comment:
This snippet below can be rewritten like so:
```
user, pass, err := getCookieCreds()
if err != nil {
    return fmt.Errorf("unable to retrieve RPC credentials from cookie: %v", err)
}

conf.RPCUser, conf.RPCPass = user, pass
return nil
```

Reply:
Done.